webapp/packages/plugin-object-viewer/src/ObjectPropertiesPage/ObjectPropertyTable/Table/Table.module.css

```diff
@@ -16,6 +16,10 @@
 .dataGrid {
   width: 100%;
   height: 100%;
+
+  & [role='columnheader'] {
+    text-transform: uppercase;
+  }
```

Comment:
looks like a new feature, do we need it?

Reply:
It's not new. It's the same style as it was before refactoring, but it was applied on a component level and can't be applied the same way now